tests/subsys/app_event_manager/src/events/name_style_events.h

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#ifndef _NAME_STYLE_EVENTS_H_
+#define _NAME_STYLE_EVENTS_H_
+
+/**
+ * @brief Name style events
+ * @defgroup name_style_event Name style events
+ *
+ * The events here are prepared to check the sections sorting.
+ * We are sorting event listeners by the event name and we relay
```

Comment:
`event subscribers` - also in other places and in commit message (please keep in mind that event listeners are separate data and have a separate memory section)

Reply:
Massive fixing performed. Thanks for this catch.